decidim_app-design/app/views/public/partials/_conferences_hero.html.erb

```diff
@@ -0,0 +1,26 @@
+<section class="extended hero"
+  style="background-image:url(<%= image_path "hero-home.jpg" %>);">
+  <div class="hero__container">
+    <div class="row">
+      <div class="columns large-10 text-left">
+        <h1 class="text-highlight hero-heading mb-none">
+          IV Jornades MetaDecidim
+        </h1>
+        <h3 class="text-highlight">Vols dissenyar el Facebook de la democràcia?</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row column medium-8 text-left mt-m">
+        <div class="column medium-6">
+          <button type="button" class="button hollow light expanded">31 NOV - 1 DIC Barcelona</button>
+        </div>
+        <div class="column medium-3">
+          <button type="button" class="button button--sc expanded">Registrate</button>
```

Comment:
please, develop the action that must be executed when you click on 'Register' button.

- Scroll down to some section?
- Open a pop up with register form?
- Go to another page?

Also, We need design for the registration page/section

Reply:
cc/ @furilo

Reply:
Ok, so the answer, in the meantime for the next iteration, is the first one: scroll down